org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/RuntimeProjectDescriptor.xtend

```diff
@@ -528,4 +531,130 @@ class RuntimeProjectDescriptor extends TestedProjectDescriptor {
 		#{"**/*.xtend","**/*.mwe2"}
 	}
 	
+	/**
+	 * @since 2.11
+	 */
+	def private getWfLaunchConfigFile() {
```

Comment:
as far as I know, you have to look at the error markers in the generated Java files :)
anyway, it OK to merge for me